lib/color.ml

```diff
@@ -0,0 +1,14 @@
+type color = int * int * int
+
+(** RGB code for black *)
+let black = (0, 0, 0)
+
+(** RGB code for white *)
+let white = (255, 255, 255)
```

Comment:
Would it be useful to provide some commonly used colours such as Red, Blue, Green, etc. in the Color module itself?

Reply:
Good idea, on it !